numpy/core/tests/test_umath.py

```diff
@@ -268,17 +268,20 @@ def floor_divide_and_remainder(x, y):
     return (np.floor_divide(x, y), np.remainder(x, y))
 
 
+def _signs(dt):
+    if dt in np.typecodes['UnsignedInteger']:
+        return (+1,)
+    else:
+        return (+1, -1)
```

Comment:
Darn, I'd missed that this was in two different files, which makes extracting it a little less useful.  I guess this is still a minor improvement though